general/platforms.rst

```diff
@@ -38,6 +38,11 @@ please refer to the file MAINTAINERS_ for contact details for various platforms.
      - Yes
      - Yes
 
+   * - `Broadcom ns3  Board <http://www.broadcom.com>`_
```

Comment:
nit: There are two spaces between "ns3" and "Board". By the way, is there a more specific URL taking someone directly to the board page? I guess no, since you've marked it as not publicly available (still had to ask).

Reply:
Thanks.  Yes I added specific reference to the SOC variant and updated. The Hardware is not public.